### PR TITLE
Re-enable Prefast for x64 CMake

### PR DESCRIPTION
**Pull request by elizabethmorrow**
Thanks for your interest in contributing to CMake!  The GitHub repository
is a mirror provided for convenience, but CMake does not use GitHub pull
requests for contribution.  Please see

  https://gitlab.kitware.com/cmake/cmake/-/tree/master/CONTRIBUTING.rst

for contribution instructions.  GitHub OAuth may be used to sign in.

### CODE REVIEW

**Review by moyo1997 — APPROVED** [review]

File: VSInsertion/Pipelines/insertion.yml [hunk 1]
```diff
@@ -1,612 +0,0 @@
...
-            SourceFolder: $(Build.SourcesDirectory)\VSInsertion\Signing
-            TargetFolder: $(SigningSolutionRoot)
-            CleanTargetFolder: true
-        - task: PowerShell@2
-          displayName: Update Commit Hash in Cgmanifest.json
-          inputs:
-            targetType: inline
-            script: (gc cgmanifest.json) -replace '\$CommitHashPlaceholder\$', (git rev-parse HEAD) | Out-File -encoding ASCII cgmanifest.json
-            workingDirectory: $(Build.SourcesDirectory)/VSInsertion/Packaging
-        - task: CopyFiles@2
```
Comment (gcampbell-msft) [inline VSInsertion/Pipelines/insertion.yml:164]: Why is all of this removed?

**Review by gcampbell-msft — COMMENTED** [review]
This looks good to me overall, but why are we removing all of that stuff from the `insertion.yml`?

**Review by gcampbell-msft — APPROVED** [review]

File: VSInsertion/Pipelines/insertion.yml (hunk 1 above)
Comment (elizabethmorrow) [reply VSInsertion/Pipelines/insertion.yml:164]: insertion.yml is no longer referenced by any pipelines, this is a vestige from the 1ES conversion effort, removing to simplify.

**Comment by elizabethmorrow** [comment]
> This looks good to me overall, but why are we removing all of that stuff from the `insertion.yml`?

Discussed offline, insertion.yml isn't used anymore. Deleting to clean up the repo.

**Pull request merged**